covert/__main__.py

```diff
@@ -109,11 +110,20 @@ def argparse():
   elif next((k for k, v in encargs.items() if av[0] in v), None) is not None:
     args.mode, ad, modehelp = 'enc', encargs, enchelp
     av.insert(0, None)
+  elif av[0].startswith('-'):
+    if av[0][:2] == '-e':
+      args.mode, ad, modehelp = 'enc', encargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
+    elif av[0][:2] == '-d':
+      args.mode, ad, modehelp = 'dec', decargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
+    else:
+      stderr.write(' 💣  Invalid or missing command (enc/dec/benchmark).\n')
+      sys.exit(1)
+    av[0] = f'-{av[0][2:]}'
+    aiter = iter(av)
```

Comment:
I got an idea from your implementation.

You can use
```python
if av[0].startswith("-e") or av[0].startswith("-d"):
  # Separate mode selector from other arguments
  av.insert(1, f'-{av[0][2:]}')
  av[0] = av[0][:2]
```

I think if you do this before line 103, the old existing handling should do everything we need. In any case, you can this way avoid special handling of `aiter`. Check if this is correct (I did not think it through entirely).